src/nextjs/index.ts

```diff
@@ -73,6 +73,8 @@ export class OttofellerNextjsProject extends NextJsTypeScriptProject implements
       tsconfig: {
         compilerOptions: {
           baseUrl: './',
+          module: 'CommonJS',
```

Comment:
@gvidon Here I tried to preserve the previous behaviour. _projen_ [recently changes it.](https://github.com/projen/projen/pull/3241). I wonder if we really need to keep this compatibility. I checked with the new defaults and project operated normally, though I'm not sure how more complicated projects might be affected. Do you think it's worth it keeping this compatibility with previous templates version, or we better go forward (anyway these options can be set manually in case of a blunder)?